block_0001/paanther-20191029162842.xml

```diff
@@ -1,17 +0,0 @@
-      <blockquote>
-        <p>At least in modern Western political thought, a socially progressive/economically conservative (everything is allowed, but the government pays for none of it? Pretty much Libertarian, correct?) is relatively rare.</p>
-      </blockquote>
-      <p>I would contend that it’s rare to see people explicitly support this ideology, but only because it’s kind of closer to the center point that people are actually defecting from. IMO almost no one who’s a social progressive and economic conservative would look at 2019 and say “tear it all down”, they’d look at it and say “We’re doing pretty okay here, guys. I have lots of issues: there are some ideas I want to implement, some people I want to get in charge, let’s get em in there, but all-in-all, I feel pretty well-represented by the powers that are and pretty hopeful about the way things are going“. These people are the people that revolutionaries <em>hate</em> - people with the conviction that things are getting better and not worse, and that continued minor changes and getting the right people in charge will make things better.</p>
-      <p>So I think it’s pretty hard to be <em>so</em> socially progressive or economically conservative that you really become what I’d call an extremist, someone who is willing to tear it all down to spite the elites. If you’re gonna be a real revolutionary, you mostly have to disagree violently with at least one of these. It’s hard for me to imagine someone who’s so positive about cutting welfare that they’d really consider threatening extralegal violence to achieve it, you know? I’d go out on a limb and say that right-wingers threatening violence about social issues is <em>way</em> more common. And how into social progressivism would you have to be to argue for actually overthrowing the government via violent means? I think Communism is probably more common than that (though to be clear, I don’t think either’s very common). Extreme environmentalism is probably much more common than both - but this also fits the pattern of mixing very well with left and right-wing worldviews.</p>
-      <p>And in a sense, the fact that this “libertarian/liberal” ideology is not ripe for extremism is because it’s the fertile ground from which dissent springs - because it’s so dominant. Academics criticize it from either direction because the point is to direct positive change, right? I think we just don’t see it praised so much because it’s the water we swim in.</p>
-      <p>I think it tends to be that most Americans are strongly liberal in the individualist sense, and this pushes them to that position of being an economic conservative or centrist and a social liberal. And since this is an ever-present background assumption, the only revolutionaries who disagree hard enough are those who disagree with one or maybe even both planks of it. (Which I guess is a kind of incentive for them to disagree with both.)</p>
-    </div>
-  </body>
-</post>
```

Comment:
I think this person has actually deleted their account, which means this one's up for grabs, so to speak - keep it around!

Reply:
It looks to me like they deleted both their account and their posts. You can delete your account but not your posts, which looks like the parent to the one above [here](https://www.reddit.com/r/TheMotte/comments/dkvpb6/culture_war_roundup_for_the_week_of_october_21/f5aizlm/).